tests/Feature/HttpConnectionTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Cookie\CookieJar;
+use GuzzleHttp\Psr7\Utils;
+use Symfony\Component\Process\PhpProcess;
+
+$process = null;
+beforeAll(function () use (&$process) {
+    $code = file_get_contents(__DIR__ . '/Stub/HttpServer.php');
+    $process = new PhpProcess($code);
+    $process->start();
+    sleep(1);
+});
+
+afterAll(function () use (&$process) {
+    $process->stop();
+});
```

Comment:
I think that we don't need to start the HttpServer with processes.
This will be more useful to run more future tests to the server.